### PR TITLE
Add notification in PR

### PR DESCRIPTION
**Pull request by jasonwzm**

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://628bfbc203efd4009595e207--cozy-fox-0defec.netlify.app

**Review by tore-statsig — COMMENTED** [review]
reviewed

File: .github/workflows/preview.yaml [hunk 1]
```diff
@@ -22,15 +22,16 @@ jobs:
...
       - name: Deploy to Netlify
         uses: nwtgck/actions-netlify@v1.2
+        id: netlify
         with:
           publish-dir: './build'
           enable-pull-request-comment: true
           enable-commit-comment: true
           overwrites-pull-request-comment: true
           deploy-message: "Preview Deploy for ${{ github.event.pull_request.title }}"
-          github-token: ${{ secrets.ROIM }}
```
Comment (tore-statsig) [inline .github/workflows/preview.yaml:31]: lol ROIM is still there

**Review by tore-statsig — APPROVED** [review]
neato

**Pull request merged**